handler.h

```diff
@@ -112,6 +112,19 @@ class NotFoundHandler : public RequestHandler {
 
 REGISTER_REQUEST_HANDLER(NotFoundHandler);
 
+class BlockingHandler : public RequestHandler {
+public:
```

Comment:
to test if multithreaded since each of our requests are handled so quickly. This is an improvement on just adding a sleep to one of the other handlers since we don't have to worry about uncommenting and commenting code. Should help with the integration test